test/rust-sample-package/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "rust-sample-package"
+version = "0.1.0"
+authors = ["Nikos Koukis <nickkouk@gmail.com>"]
```

Comment:
Just curious, whose name is this and how did it land here? I feel hesitant to use the name of someone who's not involved in the project.

Reply:
I [copied](https://github.com/colcon/colcon-cargo/blob/main/test/rust-sample-package/Cargo.toml) the package from `colcon-cargo` hence kept the original attribution.
It is true that at the end of the day it's just an empty template where the only code contribution was made by me though (adding code that passes / fails test)

Reply:
I see, it's an interesting dilemma. On one hand, I wouldn't want to use someone's contact info without their explicit consent. On the other hand, it could be interpreted as an act of attribution, as you're saying. 

Since we're using the same code in the same way that the original author intended it, just in a different (but strongly related) package, I suppose I don't object to keeping it as-is. But if anyone has a different perspective, I'd appreciate hearing it. 

I'll tag @bergercookie and maybe if he sees the notification, he can give a 👍/👎 on the usage.

Reply:
Since this sample package is not meant to be published as is, I'd change the author and email to something non-specific (Test McTestface <notareal@email.address>) to avoid attribution, but also prevent @bergercookie getting spammed. I'd do the same for `package.xml`.

Reply:
ttps://github.com/colcon/colcon-ros-cargo/pull/22/commits/a62ff2221b0038b922d99702c9b2dd66bce91289